examples/walkthroughs/trader/buysell.md

```diff
@@ -41,9 +41,9 @@ A level defines a [layer](https://en.wikipedia.org/wiki/Layering_(finance)) that
 
 ## Run Kelp
 
-Assuming your botConfig is called `trader.cfg` and your strategy config is called `buysell.cfg`, you can run `kelp` with the following command:
```

Comment:
let's keep the walkthrough files (`buysell.md`, `sell.md`, `balanced.md`) consistent w.r.t. keeping/removing the following text:
```
you can run `kelp` with the following command
```

your PR only removed it from this file

Reply:
That removal was supposed to be a revert to the original text when I simplified my change. I'll put it back here instead of removing it from the others.